Documentation/devel/inspect-containers.md

```diff
@@ -0,0 +1,354 @@
+root     11021  0.0  0.0  62280  7392 ?        Ss   17:17   0:00              \_ /usr/lib/systemd/systemd --default-standard-output=tty --log-target=null --show-status=0
+root     11022  0.0  0.0  66408  8812 ?        Ss   17:17   0:00                  \_ /usr/lib/systemd/systemd-journald
+root     11026  0.0  0.0   1212     4 pts/0    Ss+  17:17   0:00                  \_ /bin/sh
+```
+
+It's 11026!
+
+Let's start by having a look at its namespaces:
+
+```
+$ sudo ls -l /proc/11026/ns/
+total 0
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 cgroup -> 'cgroup:[4026531835]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 ipc -> 'ipc:[4026532764]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 mnt -> 'mnt:[4026532761]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 net -> 'net:[4026532702]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 pid -> 'pid:[4026532765]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 pid_for_children -> 'pid:[4026532765]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 user -> 'user:[4026531837]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 uts -> 'uts:[4026532763]'
+```
+
+We can compare it with the namespaces on the host
+
+```
+$ sudo ls -l /proc/1/ns/
+total 0
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 cgroup -> 'cgroup:[4026531835]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 ipc -> 'ipc:[4026531839]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 mnt -> 'mnt:[4026531840]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 net -> 'net:[4026532009]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 pid -> 'pid:[4026531836]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 pid_for_children -> 'pid:[4026531836]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 user -> 'user:[4026531837]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 uts -> 'uts:[4026531838]'
+```
+
+We can see that the cgroup and user namespace are the same, since rkt doesn't use cgroup namespaces and user namespaces weren't enabled for this execution.
+If, for example, we run rkt with `--net=host`, we'll see that the network namespace is the same as the host's.
+
+We can also see some interesting data about the process:
+
+```
+$ sudo cat /proc/11026/status
+Name:	sh
+Umask:	0022
+State:	S (sleeping)
+...
+CapBnd:	00000000a80425fb
+...
+NoNewPrivs:	0
+Seccomp:	2
+...
+```
+
+This tells us the container is not using the `no_new_privs` feature, but it is using [seccomp][seccomp].
+
+We can also see what capabilities are in the bounding set of the process, let's decode them with `capsh`:
```

Comment:
capabilities -> link to the capability documentation that you patched recently ;)